### ISSUE TITLE
"Index out of range error" with private trackers in Prowlarr

### ISSUE THREAD

**Issue by Made1908** [mhdzumair/MediaFusion#400]
I tried using mediafusion version 4.2.1 with private trackers in prowlarr. But whenever I tried to scrape for a movie in Stremio I don´t get any results and the following Debug error message:

ERROR::23-Dec-24 13:48:14::/mediafusion/scrapers/base_scraper.py::1403 - Error processing search result: list index out of range
Traceback (most recent call last):
  File "/mediafusion/scrapers/base_scraper.py", line 1265, in process_stream
    parsed_data = await self.parse_indexer_data(
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/mediafusion/scrapers/prowlarr.py", line 292, in parse_indexer_data
    download_url = await self.get_download_url(indexer_data)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/mediafusion/scrapers/base_scraper.py", line 1411, in get_download_url
    torrent_type = self.get_torrent_type(indexer_data)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/mediafusion/scrapers/prowlarr.py", line 88, in get_torrent_type
    flag = item.get("indexerFlags", ["public"])[0]
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^
IndexError: list index out of range

Public trackers are working fine. Let me know if you need additional information or debug logs.

**Comment by mhdzumair** [comment]
what is the indexer name?

**Comment by mhdzumair** [comment]
Connect with me in TG, https://t.me/mediafusion_stremio_addon

**Comment by mhdzumair** [comment]
Fixed in 4.3.0
